utils/node_local_test.py

```diff
@@ -1112,7 +1112,7 @@ def __str__(self):
 
         return f'DFuse instance at {self.dir} ({running})'
 
-    def start(self, v_hint=None, single_threaded=False, use_oopt=False):
+    def start(self, v_hint=None, single_threaded=False, use_oopt=False, log_mask=None):
```

Comment:
If it's an object attribute you can just set it rather than pass it in here, you don't need to do both.  If you're just passing it in here then you don't need to set a value on L1099.

My preference is probably to use object attributes rather than method parameters.

Reply:
ok ill clean it up in a follow on patch that tests only NLT to avoid re-pushing this for a full run

Reply:
fixed here https://github.com/daos-stack/daos/pull/11602